tests/test_land_vision.py

```diff
@@ -19,3 +25,36 @@
 #     print ('ret', json.dumps(ret['tile'], indent = 2))
 
 #     # _land_vision.seeLand(imageUrl)
+
+def test_urban_tree_detection_inference():
+    img_path = "./uploads/images/naip_image.png"
+    weights_dir = "./land_vision/urban_tree_detection/pretrained"
+    min_distance = 1
+    threshold_abs = None
+    threshold_rel = 0.2
+    max_distance = 10
+
+    img = cv2.imread(img_path)
+    img_rgb = cv2.cvtColor(img, cv2.COLOR_BGR2RGB)  
+    images = img_rgb[None,:,:,:]
+    
+    preprocess = eval(f'preprocess_RGB')
+    training_model, model = SFANet.build_model(
+        images.shape[1:],
+        preprocess_fn=preprocess)
+
+    weights_path = os.path.join(weights_dir,'weights.best.h5')
+    training_model.load_weights(weights_path)
+
+    preds = model.predict(images,verbose=True,batch_size=1)[...,0]
+    results = predict(preds=preds,
```

Comment:
In general I'd suggest keeping the file prefix or using a name that is less generic than `predict` so it is clear where this comes from (without having to look elsewhere in the file) and there is low chance of name conflicts.

Reply:
modified it to "get_pred_locs"